lib/error-message.js

```diff
@@ -29,19 +29,19 @@ class CustomError extends Error {
 function report(options, err, defaultPath) {
   if (options.report === 'json') {
     return Anonymize.pathsAndVersions(
-      options,
-      Anonymize.pathsAndVersions(
-        options,
-        JSON.stringify(
-          formatJson(options.debug, err, defaultPath),
-          null,
-          options.debug || options.forTests ? 2 : 0
-        )
-      )
+      JSON.stringify(
+        formatJson(options.debug, err, defaultPath),
+        null,
+        options.debug || options.forTests ? 2 : 0
+      ),
+      options.forTests
     );
   }
 
-  return Anonymize.pathsAndVersions(options, formatHuman(options.debug, err));
+  return Anonymize.pathsAndVersions(
+    formatHuman(options.debug, err),
+    options.forTests
+  );
```

Comment:
Note to self: reevalute anonymizing in this function. It's nice to have it, but we end up needing to normalize elsewhere anyway, so perhaps we could simplify this logic by simply removing it.

Reply:
> I'm guessing we can do a find+replace in the output in a helper from our test suite.

Yeah, we have to do that anyway because this doesn't fully anonymize all error paths, so I figured we might as well drop this (which simplifies some of the windows logic).